lib/discourse_api/api/topics.rb

```diff
@@ -85,6 +85,10 @@ def change_owner(topic_id, params = {})
 
         post("/t/#{topic_id}/change-owner.json", params)
       end
+
+      def topic_set_user_notification_level(topic_id, notification_level)
```

Comment:
Same as my comment above for this one. We still want the params hash as the 2nd param and we don't want use query params to make the post request.